core/shared/src/engine.cpp

```diff
@@ -93,7 +98,9 @@ ConVarHandle Engine::GetConVarHandle(std::string scvar)
 }
 
 DLLNETWORK Engine *engine = NULL;
-
+#ifdef __linux__
+int master_fd,slave_fd;
```

Comment:
@SlawekNowy Are these still needed or can they be removed?

Reply:
You can remove this